### DISCUSSION TITLE
Attempt to assign property "responsible" on null

### DISCUSSION THREAD

**Discussion by Arnaud-Askee** [dbarzin/mercator#133] [Q&A]
Bonjour,
Je démarre tout juste ma prise en main de Mercator : merci d'avance de votre patience :)

J'ai un plantage lors de l'ajout d'une nouvelle application.
J'ai renseigné tous les champs obligatoires. Les listes étaient vides : j'ai donc renseigné puis sélectionné la valeur saisie.

Je suis étonné que la liste des responsables soit vide : Où devrait elle être renseignée ? 
Je m'attendais que ce soit dans les entités (avec un type d'entité ?). Là aussi, la liste est vide alors que j'imaginais une liste pré-défini.

Merci de votre aide 

![Capture d’écran 2022-06-14 151731](https://user-images.githubusercontent.com/107484142/173586564-4c19de95-f996-449d-be8c-a548a4c4009a.png)

**Comment by dbarzin** [comment]
OK, c'est corrigé.

**Reply by Arnaud-Askee** [reply]
Super :)
Sur mon serveur, que puis-je faire pour avoir accès à cette correction ?

**Reply by dbarzin** [reply]
Pour mettre à jour : 

    cd /var/www/mercator
    git pull
